osr/interfaces/balloontransport.simba

```diff
@@ -0,0 +1,143 @@
+{$DEFINE RELD_BALLOON_INCLUDED}
+{$IFNDEF WL_OSR}
+  {$I WaspLib/osr.simba}
+{$ENDIF}
+
+type
+  ERSBalloonButton = (GRAND_TREE_BALLOON, CASTLE_WARS_BALLOON, ENTRANA_BALLOON,
+                      TAVERLEY_BALLOON, CRAFTING_GUILD_BALLOON, VARROCK_BALLOON,
+                       CLOSE_BALLOON);
+
+  TRSBalloonTransport = type TRSInterface;
+
+procedure TRSBalloonTransport.Setup(); override;
+begin
+  inherited;
+  ButtonColors := [CTS1(8564948, 0)];
+end;
+
+procedure TRSBalloonTransport.SetupAlignment(mode: ERSClientMode); override;
```

Comment:
I don't really understand this part but it works in all three modes.}

Resize classic
![image](https://github.com/Torwent/WaspLib/assets/16868241/f4102986-1f28-4bfb-8fd0-4b00e89a50b0)

Resize modern
![image](https://github.com/Torwent/WaspLib/assets/16868241/243f183c-ebc1-4ace-8e48-37db1bb178e4)

Fixed
![image](https://github.com/Torwent/WaspLib/assets/16868241/d0755757-93a3-4b52-98dc-492749e76041)